wordpress-mobile/WordPress-iOS
# Quick Start v2: Update completion logic

Refs #10860

For QS v2 the completion logic must consider both lists. Furthermore, both lists should still be displayed even when both lists are complete.

<img width="407" alt="screen shot 2019-02-07 at 10 25 47 pm" src="https://user-images.githubusercontent.com/517257/52500010-bc3edf00-2b91-11e9-81a5-0cfb254f0154.png">


Note: until the `ellipsis` menu is implemented, quick start checklists will always be visible in v2.

To test:
- create a new site (otherwise the 'create a site' item won't appear) **OR** open a site that previously started (or completed) quick start
- ensure that the checklists appear correctly when completed
- ensure both lists remain when all tours have been completed
